example/example.js

```diff
@@ -4,19 +4,21 @@ var chart;
 var selectionInfo  = d3.select('#selection-info').append('ul');
 var activitiesInfo = d3.select('#selection-info').append('dl');
 
-var clearBrushButton = document.getElementById('clearBrush');
-clearBrushButton.onclick = function clearBrush () {
+var clearButton = document.getElementById('clear');
+clearButton.onclick = function clear () {
   selectionInfo.html('');
   activitiesInfo.html('');
-  chart.clearBrush();
+  chart.clear();
+  hideClear();
 };
 
-function updateSelectionInfo (timeRange) {
-  selectionInfo.html('');
-  selectionInfo.append('li').text('Start date: ' + timeRange[0]);
-  selectionInfo.append('li').text('End date: ' + timeRange[1]);
+function toggleClearButton (visible) {
+  document.getElementById('clear').style.display = visible ? 'block' : 'none';
```

Comment:
Maybe replace this line by:

```
var currentDisplay = document.getElementById('clear').style.display;
currentDisplay = (currentDisplay === 'block' ? 'none' : 'block');
```

And then we initialize it with `display: none` or something?